elements/lisk-p2p/test/integration/discovery.ts

```diff
@@ -182,6 +182,7 @@ describe('Peer discovery', () => {
 				os: platform(),
 				height: 10000,
 				nonce: `404`,
+				advertiseAddress: true,
```

Comment:
You don't need this since it's the default value and optional in the config interface.

Reply:
Same happens in other places.